scripts/hail_batch/hgdp1kg_tobwgs_pca_nfe_new_variants_no_outliers/hgdp_1kg_tob_wgs_nfe_pca_no_outliers.py

```diff
@@ -29,8 +29,14 @@ def query():
         (mt.hgdp_1kg_metadata.population_inference.pop == 'nfe')
         | (mt.s.contains('TOB'))
     )
-    # remove outlier samples TOB1734 and TOB1714
-    mt = mt.filter_cols((mt.s != 'TOB1734') & (mt.s != 'TOB1714') & (mt.s != 'TOB1126'))
+    # remove outlier samples
+    mt = mt.filter_cols(
```

Comment:
Does this work?

```suggestion
    mt = mt.filter_cols(mt.s not in ['TOB1734', 'TOB1714', 'TOB1126', 'TOB1653', 'TOB1668'])
```

Reply:
I don't think so. I tested it and got the error:
`TypeError: 'Expression' objects have no static length: use 'hl.len' for the length of collections`